Update windows-security-baselines.md (#6484)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-security-baselines.md

> -   [Microsoft Endpoint Configuration Manager](https://www.microsoft.com/cloud-platform/system-center-configuration-manager)
> -   [Operations Management Suite](https://www.microsoft.com/cloud-platform/operations-management-suite)

Reviewer: (same here, whitespace code style consistency)
```suggestion
- [Microsoft Endpoint Configuration Manager](https://www.microsoft.com/cloud-platform/system-center-configuration-manager)
- [Operations Management Suite](https://www.microsoft.com/cloud-platform/operations-management-suite)
```

- [Configuration Management for Nano Server](https://docs.microsoft.com/archive/blogs/grouppolicy/configuration-management-on-servers/)

> -   [Microsoft Security Guidance Blog](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/bg-p/Microsoft-Security-Baselines)
> -   [Microsoft Security Compliance Toolkit Download](https://www.microsoft.com/download/details.aspx?id=55319)
> -   [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=55319) 

Reviewer: ```suggestion
- [Microsoft Security Guidance Blog](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/bg-p/Microsoft-Security-Baselines)
- [Microsoft Security Compliance Toolkit Download](https://www.microsoft.com/download/details.aspx?id=55319)
- [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=55319) 
```
Whitespace adjustment, for code style consistency.

-   [Microsoft Endpoint Configuration Manager](https://www.microsoft.com/cloud-platform/system-center-configuration-manager)
-   [Operations Management Suite](https://www.microsoft.com/cloud-platform/operations-management-suite)

> -   [Configuration Management for Nano Server](https://blogs.technet.microsoft.com/grouppolicy/2016/05/09/configuration-management-on-servers/)

Reviewer: Since you are already doing this, please note this change as well:
```suggestion
- [Configuration Management for Nano Server](https://docs.microsoft.com/archive/blogs/grouppolicy/configuration-management-on-servers/)
```
